Example/src/ChatHeadsExample.tsx

```diff
@@ -96,9 +106,13 @@ function ChatHeads({ children }) {
       </PanGestureHandler>
     </>
   );
-}
+};
 
-function Followers({ transX, transY, children }) {
+type FollowersProps = {
+  readonly transX: Animated.SharedValue<number>;
```

Comment:
It's coming from `useDerivedValue` which uses `Readonly<Animated.SharedValue<T>>`. I guess it doesn't change much. FWIW it can be used to show that we don't change provided shared values.